src/devices/Seesaw/SeesawModules.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Seesaw
+{
+   public partial class Seesaw : IDisposable
```

Comment:
rename the namespace or the class name, otherwise it will be really annoying to use (you will need to write both each time this is used)

Reply:
@krwq 
Isn't this what the other devices have done though for example -

```
namespace Iot.Device.Mcp3008
{
    /// <summary>
    /// MCP3008 Analog to Digital Converter (ADC)
    /// </summary>
    public class Mcp3008 : IDisposable
    {
```